jestsetup.js

```diff
@@ -0,0 +1,25 @@
+// Make Enzyme functions available in all test files without importing
+/* eslint-disable import/no-extraneous-dependencies */
+import Enzyme, { mount, render, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+global.shallow = shallow;
+global.render = render;
+global.mount = mount;
+
+// TODO: Test running jest with this removed and see if tests still work
+// More info: https://github.com/cerner/terra-react-svg-loader/pull/1#discussion_r100876464
+// Skip createElement warnings but fail tests on any other warning
+// eslint-disable-next-line no-console
+console.error = (message) => {
+  if (!/(React.createElement: type should not be null)/.test(message)) {
+    throw new Error(message);
+  }
+};
+
+window.matchMedia = () => ({ matches: true });
```

Comment:
We should be able to drop this. This was added specifically for [terra-time-input](https://github.com/cerner/terra-framework/blob/4880772b8b92bb05f99ef3f3084a8d1e4c389bdc/packages/terra-time-input/src/TimeUtil.js#L280) and [terra-date-input](https://github.com/cerner/terra-framework/blob/ca8fde828d309978e69ab801d59044f5e71626b0/packages/terra-date-input/src/DateInput.jsx#L519).